test/TensorFlow/integration.swift

```diff
@@ -228,6 +228,65 @@ public func test_bool_param2(cond: Bool, // expected-warning {{'cond' implicitly
 // CHECK-NEXT: [[PROGRAM:%.*]] = apply [[STARTFN:%.*]](
 // CHECK: cond_br [[BOOLVAL]],
 
+public func test_multiple_ifs(status: Bool // expected-warning {{'status' implicitly copied to the accelerator}}
```

Comment:
To put the expected-warning on top and put the `) {` back on this line, you can write:
```swift
// expected-warning @+1 {{'status' implicitly copied to the accelerator}}
public func test_multiple_ifs(status: Bool)
``

Reply:
Ah, that is nice. Thanks for the tip. :)